main/admin/index.php

```diff
@@ -26,6 +26,21 @@
 
 $nameTools = get_lang('PlatformAdmin');
 
+if (api_is_multiple_url_enabled()) {
+    $accessUrlId = api_get_current_access_url_id();
+
+    if ($accessUrlId != -1) {
+        $urlInfo = api_get_access_url($accessUrlId);
+        $url = api_remove_trailing_slash(preg_replace('/https?:\/\//i', '', $urlInfo['url']));
+        $cleanUrl = replace_dangerous_char($url);
```

Comment:
I believe the URL is safe anyway (it comes from the access_url table, and all entries to that table are filtered), so no need to replace_dangerous_char() here.